internals/reaper/reaper.go

```diff
@@ -75,22 +73,28 @@ func Stop() error {
 	started = false
 	mutex.Unlock()
 
+	err := setChildSubreaper(0)
```

Comment:
Would it be theoretically safer to disable receiving ```unix.SIGCHLD``` signals, before we terminate the processing loop that deals with them ?